onnx/inliner/inliner.cc

```diff
@@ -448,8 +413,11 @@ void InlineFunctions(ModelProto& model, FunctionMap map, bool convert_version) {
     if (iter != map.end()) {
       callee = *iter->second.first;
       int64_t target_version = iter->second.second;
-      // Rename and specialize called function body
-      Specializer::Specialize(node, callee, "__" + std::to_string(++inline_count), name_generator);
+      // Bind attribute parameters
+      internal::AttributeBinder::BindAttributes(node, callee);
+
+      // Rename variable names in callee
```

Comment:
Maybe describe the renaming logic to help readers?

Reply:
See lines 135-140.